### ISSUE TITLE
Getting error: Cannot read property 'gain' of null when using the latest 2.0 beta

### ISSUE THREAD

**Issue by alvinsight** [goldfire/howler.js#511]
Hi guys, 
Thanks for all the work on Howler v2, it seems a lot better and more reliable, I'm loving it so far.
Since I upgraded from beta8 to the latest dev commit I'm getting the following error : 
```
Cannot read property 'gain' of null
```
When I call Howler.mute(false), the error is coming from here: 
``` 
// With Web Audio, we just need to mute the master gain.
if (Howler.usingWebAudio) {
	 Howler.masterGain.gain.value = muted ? 0 : self._volume;
}
```

**Comment by goldfire** [comment]
What browser/platform?

**Comment by alvinsight** [comment]
Hi James,

On windows, on any browser since upgrading from beta8.
Thanks,
Alvin

**Comment by goldfire** [comment]
Can you give a test case? I've tested on Chrome and Edge and am unable to reproduce an error.

**Comment by ray007** [comment]
I'm getting this error as well, but not every time.
There seems to be some unlucky timing involved, don't yet know more.

I'm avoiding the problem for now by setting `Howler.usingWebAudio = false` before calling any Howler method.

**Comment by Willian-Zhang** [comment]
actually this happends here with Chrome:
`http://drrr.com/`
actions to reproduce:
choose a username to login => create a 'room' with `audio sharing` enabled => the error is reproduced as soon as you 'got in the room'

**Comment by goldfire** [comment]
Thanks for the additional info, this should be fixed with 83906a1ce4df9e65d4b2e3700ca53c7a58c450ef and will be in b13 shortly.
